programming_examples/basic/matrix_multiplication/whole_array/Makefile.chess

```diff
@@ -11,13 +11,15 @@ srcdir := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 subdir=whole_array
 targetname=whole_array
 
-M?=640
-K?=896
-N?=768
-m?=16
-k?=32
-n?=48
-n_aie_cols?=2
+M?=128
+K?=176
+N?=128
```

Comment:
It would be good to have another larger size GEMM in test, here too.

Reply:
Same here.